### PR TITLE
add v4 release

### PR DESCRIPTION
**Pull request by kgaonkar6**
#### What scientific question is your analysis addressing?
v4 release

### CODE REVIEW

File: doc/data-formats.md [hunk 1]
```diff
@@ -90,158 +24,38 @@ For each method/measure (e.g., RSEM TPM, RSEM isoform counts), there are two mat
...
-The identifiers in the `Kids_First_Biospecimen_ID` column of the `pbta-histologies.tsv` file will be non-overlapping for different experimental strategies, as these identifiers map to a _library or assay_.
-**Analysts should use the identifiers in the `sample_id` to connect DNA-seq and RNA-seq assays from the same sample.**
-
-For an example, see [the `sample_id` mapping step of the OncoPrint pipeline](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/oncoprint-landscape/00-map-to-sample_id.R).
-
-Note that some individual participants (tracked via the `Kids_First_Participant_ID` column) will have multiple samples included in the PBTA dataset.
-Please [see the independent specimens files section](#independent-specimen-lists) for more information.
-
-<!-- TODO: add a notebook with an example for how to map between identifiers -->
+* `kfnbl-histologies.tsv`
```
Comment (jharenza) [inline doc/data-formats.md]:
```suggestion
* `histologies.tsv`
```

**Review by jharenza — COMMENTED** [review]
just going to swap out the release notes file for the update i made

**Review by jharenza — COMMENTED** [review]
download works for me, but would like others to review since I had added and updated files :)

**Comment by jharenza** [comment]
@ewafula and @logstar will one or both of you test this download and approve the PR if all looks good on your end? then I think we can get this in.

**Review by ewafula — APPROVED** [review]
Download works for me and merged histologies file looks ok.

**Pull request merged**